Dockerfile.arm64

```diff
@@ -0,0 +1,39 @@
+FROM golang:1.22.7-bullseye AS builder
+
+WORKDIR /app
+
+# Install dependencies for ebpf compilation
+RUN apt update \
+    && apt install --no-install-recommends -y clang llvm gcc libbpf-dev \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
You deleted the apt cache here, and then populated the apt cache again on line10.